baseapp_pdf/management/commands/base.py

```
@@ -0,0 +1,31 @@
+from typing import Any
+
+from django.core.management.base import BaseCommand
+
+
+class BasePDFCommand(BaseCommand):  # pragma: no cover
+    def handle(self, *args: Any, **options: Any) -> None:
+        try:
+            self._handle(*args, **options)
+        except KeyboardInterrupt:
+            self.stdout.write("\r\n")
+        except Exception:  # NOSONAR - S5754
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)